index.js

```diff
@@ -38,18 +97,46 @@
  * And returns the first and last name concatenated together (i.e. 'Bob Evans').
  */
 
+function getFullName(name) {
+	var name = {
+		firstName: 'Bob',
+		lastName: 'Evans'
+	};
+	console.log(name.firstName + ' ' + name.lastName);
+	return name.firstName + ' ' + name.lastName;
+};
```

Comment:
I mentioned this in class, but we actually don't want to define this Object within the function itself. As written, your variable `name` will override the parameter `name`, so the result will be the same no matter what is passed into the function.

Instead what we want to do is assume the name Object is passed in:

```
function getFullName(name) {
    return name.firstName + ' ' + name.lastName;
}
```

then we can call it like:

```
getFullName({firstName: 'Jacob', lastName: 'Friedmann'});
```